spec/features/bool_values_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'rails_helper'
+
+describe "Bool values", type: :feature do
+  context "shows boolean fields" do
+    before do
+      register_index(Invoice) do
+        bool_column :paid
+      end
+    end
+
+    it "shows localized attribute true value" do
+      Invoice.create(paid: true)
+      visit admin_invoices_path
+      expect(page).to have_content 'pagada'
+    end
+
+    it "shows localized attribute false value" do
+      Invoice.create(paid: false)
+      visit admin_invoices_path
+      expect(page).to have_content 'no pagada'
+    end
+  end
+
+  context "passing a block" do
+    before do
+      register_show(Invoice) do
+        bool_row(:paid) do |invoice|
```

Comment:
Unused block argument - `invoice`. You can omit the argument if you don't care about it.